src/Development/IDE/Spans/AtPoint.hs

```diff
@@ -54,36 +53,39 @@ atPoint
   -> [SpanInfo]
   -> Position
   -> Maybe (Maybe Range, [T.Text])
-atPoint IdeOptions{..} tcs srcSpans pos = do
+atPoint IdeOptions{..} _ srcSpans pos = do
```

Comment:
If we never use the typechecked modules, there is no point in passing them as an argument.